.github/workflows/validate_schematron.yml

```diff
@@ -0,0 +1,17 @@
+name: validate schematron
+
+on: [push]
+
+jobs:
+  validate-schematron:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@main
+    - name: Enable xmllint matcher
+      uses: korelstar/xmllint-problem-matcher@v1
+    - name: Validate articles
+      uses: ChristophWurst/xmllint-action@master
```

Comment:
I would have thought that `xmllint` cannot be used to validate against modern Schematron, though. Does this work? I just tried to validate a document against dhqTEI-ready.sch using xmllint version “using libxml version 20913” on an Ubuntu 22.04 system, and got a page of over 30 error messages _from the schema_. IIRC, libxml2 never really moved from Schematron 1.x to ISO Schematron.

I do not think there is a gitHub action for it yet, but my current favorite Schematron validator is what I call `mausatron` and the rest of the world (including its author, David Maus) calls [schxslt](https://github.com/schxslt/schxslt).

Reply:
I defer to you on schematron validation! 

The github action is really just a convenience — if you can install it and run it on ubuntu, you can use it. GitHub Actions will let you execute arbitrary commands, so you could swap this out with whatever it takes to install schxslt/mausatron and then run it.

Reply:
Any chance you (@rlskoeser) would be willing to “sit down” with me (over Slack or Signal or Zoom or whatever) and help me do that?